pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -26,6 +26,12 @@ var genericPreflightChecks = [...]PreflightCheck{
 		fixDescription:   "Caching podman remote binary",
 		fix:              fixPodmanBinaryCached,
 	},
+	{
+		checkDescription: "Checking if goodhosts binary is cached",
+		check:            checkGoodhostBinaryCached,
+		fixDescription:   "Caching goodhosts binary",
+		fix:              fixGoodHostBinaryCached,
+	},
```

Comment:
Do we want  to make it mandatory by default?

Reply:
@cfergeau I am incline to make it mandatory and since this is on our crc-repo side so we have full control over it.

Reply:
I realized it can't be skipped at all, so we'll always have the 2 DNS configuration methods running in parallel. Enabling it by default or not is one thing, but I really think it should be possible to disable it entirely, at least for debugging/testing purposes. Then you will have people who don't want (can't) run a 5.5MB suid binary.